app/helpers/rails_admin/application_helper.rb

```diff
@@ -163,7 +163,14 @@ def menu_for(parent, abstract_model = nil, object = nil, only_icon = false)
               else
                 'javascript:void(0)'
               end
-            content_tag(:a, label, {href: href}.merge(action.pjax? ? {class: ['pjax']} : {}))
+
+            extra_classes = []
+            extra_classes << 'pjax' if action.pjax? && action.target_window != :_blank
+
+            extra_tags = {}
+            extra_tags[:target] = action.target_window if action.target_window.present?
+
+            content_tag(:a, label, {href: href, class: extra_classes}.merge(extra_tags))
```

Comment:
```suggestion
            content_tag(:a, label, {href: href, target: action.target_window}.merge(action.pjax? ? {class: ['pjax']} : {}))
```

Reply:
One of the issues of PJAX, is that, when enabled, it was removing the effect of the `target=_blank` for some reason